mev-boost-relay/services/api/proofs.go

```diff
@@ -50,17 +61,14 @@ func verifyInclusionProof(log *logrus.Entry, transactionsRoot phase0.Root, proof
 		}
 
 		leaves[i] = txHashTreeRoot[:]
+		indexes[i] = int(proof.GeneralizedIndexes[i])
```

Comment:
This is safe because max i64 is 9,223,372,036,854,775,807 and all generalized indexes for transactions are capped at 2^22 -1 = 4,194,303 by consensus specs

Reply:
Actually, for the multiproof verification it is required that `indexes` is a `[]int`. Given that generalized indexes will reasonably be small numbers (in the order of $2^21$ in our use case), we should not worry too much about it